src/Microsoft.AspNetCore.Sockets.Client.Http/Internal/WebSocketsTransport.cs

```diff
@@ -348,10 +348,14 @@ private static Uri ResolveWebSocketsUrl(Uri url)
             {
                 await Running;
             }
-            catch
+            catch (Exception ex)
             {
+                Log.TransportStopped(_logger, ex);
                 // exceptions have been handled in the Running task continuation by closing the channel with the exception
```

Comment:
We should make this consistent with the rest and make it never throw (tests would need to be updated)